docs/developing/publishing-apps.md

```diff
@@ -0,0 +1,105 @@
+# App publishing guide
+
+After you've completed developing and testing your app's package for Sandstorm, you can publish it in the [Sandstorm App Market](https://apps.sandstorm.io/).
+
+## Verify your identity
+
+### Sign up with [Keybase.io](https://keybase.io)
```

Comment:
Multiple people have complained about not being sure about the difference between Keybase keys and Sandstorm ed25519 keys.

I suggest writing some text to clarify this in the docs, if possible.